layers/cassandra/config.json

```diff
@@ -1,5 +1,4 @@
 {
     "php": [
-        "74"
```

Comment:
Shouldn't we have at least one valid PHP version?

Reply:
I added PHP 8.0 for cassandra and scrypt.

scrypt compiled ok, but cassandra failed:

> pecl/cassandra requires PHP (version >= 7.2.0, version <= 7.99.99), installed version is 8.0.27

I think we shouldn't block this PR for a feature that didn't exist previously, I'll revert PHP 8 support for Cassandra and just leave it as is 🤷 Maybe someone else can contribute PHP 8 support later.